google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java

```diff
@@ -173,6 +173,22 @@ public Blob copyToStrings(String bucketName, String blobName) {
   }
 
   /**
+   * Example of moving a blob to a different bucket with a different name.
+   */
+  public Blob moveTo(Blob srcBlob, String destBucket, String destBlob) {
```

Comment:
Hm... tried to create a new class, but it doesn't look like the javadoc script can handle having multiple snippets classes target a single javadoc file. Ended up just adding it to the existing snippets, and modifying the test to reset the blob between tests. PTAL.